<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DB_Form</class>
 <widget class="QMainWindow" name="DB_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="uploadBtn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>150</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteBtn">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>100</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="measurements_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>450</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="visual_table">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>450</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="naming_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>450</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="error_table">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>450</width>
      <height>320</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
